benchmarks/torchbench/voirfile.py

```diff
@@ -1,11 +1,88 @@
+    #     .augment(batch=lambda self: self.example_inputs)
+    #     .augment(batch_size=lambda batch: len(batch))
+    #     .give()
+    # )
+    # (
+    #     ov.probe("/torchbenchmark.util.framework.vision.model_factory/TorchVisionModel/train(real_input) > #endloop__ as step")
+    #     .augment(batch=lambda real_input: real_input)
+    #     .augment(batch_size=lambda batch: len(batch))
+    #     .give()
+    # )
+
+    yield ov.phases.parse_args
+
+    # model = ov.options.ARGV[0]
+    # with open(f"{os.path.dirname(__file__)}/torchbenchmark/models/{model}/__init__.py", "r") as f:
+    #     f = f.read()
+    #     for mf in _MODELS_FACTORIES:
+    #         if mf in f:
+    #             break
+    #     else:
+    #         if model == "Background_Matting":
+    #             (
+    #                 ov.probe(f"/torchbenchmark.models.{model}/Model/train(data as batch) > #endloop_data as step")
+    #                 .augment(batch_size=lambda batch: len(batch))
+    #                 .give()
+    #             )
+    #             (
+    #                 ov.probe(f"/torchbenchmark.models.{model}/Model/train > lossG as loss")["loss"]
+    #                 .map(float)
+    #                 .give("loss")
+    #             )
+    #         elif model == "dcgan":
+    #             (
+    #                 ov.probe(f"/torchbenchmark.models.{model}/Model/train(benchmark_pic as batch, errD, errG) > #endloop_i as step")
+    #                 .augment(batch_size=lambda batch: len(batch))
+    #                 .augment(loss=lambda errD, errG: float(errD + errG))
+    #                 .give()
+    #             )
+    #         elif model == "detectron2_maskrcnn":
+    #             (
+    #                 ov.probe(f"/torchbenchmark.models.{model}/Model/train(data as batch) > #endloop_data as step")
+    #                 .augment(batch_size=lambda batch: len(batch))
+    #                 .give()
+    #             )
+    #             (
+    #                 ov.probe(f"/torchbenchmark.models.{model}/Model/train > loss")["loss"]
+    #                 .map(float)
+    #                 .give("loss")
+    #             )
+    #         else:
+    #             (
+    #                 ov.probe(f"/torchbenchmark.models.{model}/Model/train(self) > #endloop__ as step")
+    #                 .augment(batch=lambda self: self.example_inputs[0])
+    #                 .augment(batch_size=lambda batch: len(batch))
+    #                 .give()
+    #             )
+    #             (
+    #                 ov.probe(f"/torchbenchmark.models.{model}/Model/train > loss")["loss"]
+    #                 .map(float)
+    #                 .give("loss")
+    #             )
+
+    yield ov.phases.load_script
+
+    ov.probe("//run_one_step > gpu_time").give()
+    ov.probe("//run_one_step > cpu_dispatch_time").give()
+    ov.probe("//run_one_step > cpu_walltime").give()
+    ov.probe("//run_one_step > tflops").give()
```

Comment:
By only keeping this, we can feed the torchbench stats for all the benches

Reply:
But I'm still working on the instrumentation of the other benches

Reply:
The code never reaches the point where those variables are set and seams to be killed before this line https://github.com/pytorch/benchmark/blob/ff7114655294aa3ba57127a260dbd1ef5190f610/run.py#L87. By checking the code and the logs, it seams that 3 steps are missing:
```
timm_efficientnet.0 Start: 2022-04-08 01:05:46.652672
timm_efficientnet.0 > Running train method from timm_efficientnet on cuda in eager mode.
timm_efficientnet.0 {'descr': 'train', 'progress': 0, 'total': 10}
timm_efficientnet.0 {'descr': 'train', 'progress': 1, 'total': 10}
timm_efficientnet.0 {'descr': 'train', 'progress': 2, 'total': 10}
^^^^^^^^^^ The steps before corresponds to the warmup phase ^^^^^^^^^^
timm_efficientnet.0 {'descr': 'train', 'progress': 3, 'total': 10}
timm_efficientnet.0 {'train_rate': 626.4610184386565}
timm_efficientnet.0 {'descr': 'train', 'progress': 4, 'total': 10}
timm_efficientnet.0 {'descr': 'train', 'progress': 5, 'total': 10}
timm_efficientnet.0 {'descr': 'train', 'progress': 6, 'total': 10}
timm_efficientnet.0 {'descr': 'train', 'progress': 7, 'total': 10}
timm_efficientnet.0 {'descr': 'train', 'progress': 8, 'total': 10}
timm_efficientnet.0 {'descr': 'train', 'progress': 9, 'total': 10}
timm_efficientnet.0 {'descr': 'train', 'progress': 10, 'total': 10}
^^^^^^^^^^ The code requests 10 iterations so it seams that 3 steps are missing ^^^^^^^^^^
timm_efficientnet.0 End: 2022-04-08 01:06:22.877753
```

It indeed seams that the execution does not complete correctly. Is it because `milabench` expects 10 steps and receives 13 instead? (printing `'total': 10` from the beginning does seam to indicate that)

Reply:
Yeah, you will probably need to remove the `--stop` option for this to work, otherwise `voir` will abort the process before it gets there.

Reply:
Ah perfect! I've just pushed it 1000 steps but I'll remove it altogether